common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -930,7 +930,10 @@ protected boolean idTokenIsNull(@NonNull final ICacheRecord cacheRecord,
     }
 
     protected Set<String> addDefaultScopes(@NonNull final TokenCommandParameters commandParameters) {
-        final Set<String> requestScopes = commandParameters.getScopes();
+        return addDefaultScopes(commandParameters.getScopes());
```

Comment:
Can we follow MSAL practice and add the default scopes in the same place as they do? Which is in the controller, not in the interface.

Reply:
Done.

Reply:
Is this still needed, now that default scopes are added in the controller?

Reply:
I have removed this change